package.json

```diff
@@ -57,37 +57,44 @@
       "default": "",
       "order": 3
     },
+    "showSpellErrorAsInfo": {
+      "title": "Show spelling errors with blue underline (requires restart)",
+      "description": "Shows spelling errors with a blue underline (info severity), rather than the orange underline (warning severity).",
+      "type": "boolean",
```

Comment:
Change to an enumeration with the values `info`, `warning` (default), `error` and call it `spellingErrorSeverity`. Update the documentation strings.